src/commands/shell.mjs

```diff
@@ -21,11 +21,11 @@ async function shellCommand(argv) {
 
   /** @type {import('node:repl').ReplOptions} */
   const replArgs = {
-    prompt: `${argv.db_path || ""}> `,
+    prompt: `${argv.database || ""}> `,
     ignoreUndefined: true,
     preview: argv.apiVersion !== "10",
     // TODO: integrate with fql-analyzer for completions
-    completer: argv.apiVersion === "10" ? () => [] : undefined,
+    completer: () => [],
```

Comment:
So, we technically do lose functionality here vs the old CLI:
![iTerm2 2024-12-09 12 00 19](https://github.com/user-attachments/assets/5852546f-a03b-4ae4-aec8-275e36bc653a)

I'm not sure I care, given that FQL v4 is on the deprecation path.

Reply:
Yeah your right - let me take a closer look

Reply:
the way the old code works is by making a nodeJS sub-vm with a context object that only contains the names defined in v4 syntax. so, setTimeout isn't a suggestion because setTimeout isn't defined as a global var in the sub-vm that evaluates completions.

https://nodejs.org/api/vm.html#scriptruninnewcontextcontextobject-options

Reply:
Okay I think v4 completions should be back -

<img width="645" alt="Screenshot 2024-12-09 at 4 40 42 PM" src="https://github.com/user-attachments/assets/ffd0b86f-16d3-481f-9aca-aceae4efce40">

Reply:
awesome, thank you for looking into it!